ExampleOutput/Commands.g.cs

```diff
@@ -47,274 +52,276 @@ public override void Validate(CommandResult commandResult)
         }
     }
 
-    public partial class StarTrekCommand : GeneratedCommandBase<StarTrekCommand.Result, FranshiseCommand>, ICommandHandler
+    public class Commands
```

Comment:
This layer is in response to a reviewer that found it awkward that the same name (like `StarTrek` appeared in two places in `AddCommand` and this naming fixed it.